### ISSUE TITLE
train result

### ISSUE THREAD

**Issue by chenyu1330** [Ouxiang-Li/SAFE#1]
testing on...
testset,accuracy,avg precision
biggan,82.45,91.11643243940034
cyclegan,95.5748865355522,99.402642430682
deepfake,70.00739644970415,97.58750969106768
gaugan,83.67999999999999,90.24643458260435
progan,99.8625,99.99234107423821
stargan,100.0,100.0
stylegan,95.39385847797062,99.93191150582645
stylegan2,98.60415623435152,99.9929293951795
mean,90.69659971219731,97.28377513987482
Thank you very much for your work. I have trained with 4-class ProGAN myself and verified that mean acc is only 90, which is very low in deepfake. What is the problem？

**Comment by Ouxiang-Li** [comment]
Thank you for your attention. It seems that your replication results differ significantly from our practice, which could be related to factors such as the running environment, experiment seeds, and others. Could you provide more detailed experimental settings for our reference? Alternatively, if you want to replicate the results in our paper, you can directly use the weights we provided in `./checkpoint/checkpoint-best.pth`.

**Comment by jackyccl** [comment]
Hi @Ouxiang-Li 
First of all, thank you so much for your fantastic work.
I have also tried to reproduce your work and found out that there was this issue too mentioning about the discrepancy of the result reproduction.
Particularly, my setting is as follows:
Ubuntu 22.04, CUDA 12.4, Nvidia A6000 Ada
numpy==1.26.4
nvidia-cublas-cu12==12.1.3.1
nvidia-cuda-cupti-cu12==12.1.105
nvidia-cuda-nvrtc-cu12==12.1.105
nvidia-cuda-runtime-cu12==12.1.105
nvidia-cudnn-cu12==8.9.2.26
nvidia-cufft-cu12==11.0.2.54
nvidia-curand-cu12==10.3.2.106
nvidia-cusolver-cu12==11.4.5.107
nvidia-cusparse-cu12==12.1.0.106
nvidia-nccl-cu12==2.19.3
nvidia-nvjitlink-cu12==12.8.93
nvidia-nvtx-cu12==12.1.105
opencv-python==4.11.0.86
packaging==25.0
pillow==11.2.1
protobuf==6.30.2
pytorch-wavelets==1.3.0
PyWavelets==1.6.0
PyYAML==6.0.2
requests==2.32.3
scikit-learn==1.6.1
scipy==1.13.1
six==1.17.0
sympy==1.14.0
tensorboardX==2.6.2.2
threadpoolctl==3.6.0
timm==0.6.12
torch==2.2.1
torchvision==0.17.1
tqdm==4.67.1
triton==2.2.0
typing_extensions==4.13.2
urllib3==2.4.0

I made no changes to the relevant codes and presumably the seeds should be as similar to what the author had set. Please will you give some feedback or if you need any more information about the environment, please feel free to reply! Appreciate it!

**Comment by Ast111** [comment]
I have the same problem, the experimental results of the deepfake dataset are much lower than the results in the paper.

**Comment by jackyccl** [comment]
Greetings @Ouxiang-Li, would you mind to look into this issue? As I found out that the learning rate mentioned to be 0.005 in the paper, however the uploaded training code was using lr of 1e-2.

**Comment by yjtlab** [comment]
> Greetings [@Ouxiang-Li](https://github.com/Ouxiang-Li), would you mind to look into this issue? As I found out that the learning rate mentioned to be 0.005 in the paper, however the uploaded training code was using lr of 1e-2.

same question, did you have ideas?
